pkg/apis/metrics/v1alpha1/appmetricsconfig_types.go

```diff
@@ -12,6 +12,9 @@ type AppMetricsConfigSpec struct {
 // AppMetricsConfigStatus defines the observed state of AppMetricsConfig
 // +k8s:openapi-gen=true
 type AppMetricsConfigStatus struct {
+
+	//app host url
+	Host string `json:"host"`
```

Comment:
Should it not obtain dynamically? Is not the goal here expose the route created/managed by the project? 
If yes, see how it was done here: https://github.com/aerogear/mobile-security-service-operator/blob/master/pkg/controller/mobilesecurityserviceapp/configmaps.go#L18

Reply:
It does obtain it dynamically (in the controller). It then sets the value of this property in the status of the CR.